notebooks/region_search/Region Searching Workbook.ipynb

```diff
@@ -2813,36 +2817,608 @@
     "del tmpdf"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "a2e81bd6",
+   "metadata": {},
+   "source": [
+    "### (RA, Dec) searching of image sets"
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": 77,
+   "execution_count": 74,
+   "id": "f3417ffd",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\"set_1\" is the first set name in overlapping_sets.\n",
+      "Its first record dataId: {instrument: 'DECam', detector: 1, visit: 898287}.\n",
+      "It has an (RA, Dec) coordinate of (351.0695696334572, -4.336293374423113).\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Fetch an (RA, Dec) pair from an image set.\n",
+    "first_set_name = list(overlapping_sets.keys())[0]\n",
+    "first_sets_data_id = overlapping_sets[first_set_name][0]\n",
+    "searching_ra_dec = df.set_index(\"data_id\")[\"center_coord\"].to_dict()[first_sets_data_id]\n",
+    "\n",
+    "print(f'\"{first_set_name}\" is the first set name in overlapping_sets.')\n",
+    "print(f\"Its first record dataId: {first_sets_data_id}.\")\n",
+    "print(f\"It has an (RA, Dec) coordinate of {searching_ra_dec}.\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 75,
    "id": "bfb23aef",
    "metadata": {},
    "outputs": [],
    "source": [
-    "def ra_dec_search_overlapping_sets(df, overlapping_sets):\n",
+    "def ra_dec_search_overlapping_sets(df, overlapping_sets, ra_dec, search_radius, verbose=False):\n",
     "    \"\"\"\n",
     "    2/6/2024 COC\n",
     "    Implementing an extremely basic (RA, Dec) query functionality.\n",
     "    This will work within the overlapping_sets framework.\n",
     "    \"\"\"\n",
-    "    pass"
+    "    ra_dec_coord = SkyCoord(ra=ra_dec[0] * u.degree, dec=ra_dec[1] * u.degree)\n",
+    "\n",
+    "    all_centers = []\n",
+    "    all_labels = []\n",
+    "\n",
+    "    id_to_coord = df.set_index(\"data_id\")[\"center_coord\"].to_dict()\n",
+    "\n",
+    "    for i, set_name in enumerate(list(overlapping_sets.keys())):\n",
+    "        all_labels.append(set_name)\n",
+    "        center_data_id = overlapping_sets[set_name][0]\n",
+    "        # print(f\"See set_name is {set_name} and center_data_id is {center_data_id}.\", flush=True)\n",
```

Comment:
Can we either remove this line or have

```
if verbose:
    print(f"...")
```

Reply:
Removed, this was for debugging. Too much spamming if we leave it in for verbose even.